src/Haskell/Ide/Engine/Server.hs

```diff
@@ -424,7 +436,9 @@ reactor inp diagIn = do
                   reactorSend $ NotShowMessage $ fmServerShowMessageNotification J.MtWarning cabalMsg
                   reactorSend $ NotLogMessage $ fmServerLogMessageNotification J.MtWarning cabalMsg
 
-            Nothing -> return ()
+            Left (_ :: Yaml.ParseException) -> do
+              logm "Failed to parse it"
+              reactorSend $ NotShowMessage $ fmServerShowMessageNotification J.MtError "Couldn't parse hie.yaml"
```

Comment:
@jneira I could not make your test happy in any other way. 
Would that be alright to already show this error in the startup? I would actually say, yes.